free-1.md

```diff
@@ -10,41 +10,83 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+_My idea would help organize to do lists. Right now, people write to do 
+lists by what to do first, which is not always an easy way of thinking 
+about it and may cause people to forget tasks because they forgot
+about a certain category in their to do list. So, another way people think of
+a to do list by those broad categories. For instance, people organize cleaning
+by room or homework by class. However, it's then hard to see the most 
+efficient way of doing the tasks or even which tasks need to be done first._
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_A language allows for a broad amount of personalization, so that this can 
+meet the needs of a large userbase. Additionally, hopefully the language 
+would be able to give the user multiple options of a to do list. If it seems 
+daunting to do certain tasks in a row,_
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+_I like that this can be a way to optimize time for a large group of people 
+who do not necessarily have a background with technology. I thought of it 
+because I ride horses with the Claremont Equestrian Team. On show days, we 
+have to get horses ready through a multitude of tasks per horse, from grooming 
+the horses to warming them up. We will often have very short, stressful 
+periods followed by long period of time with almost nothing to do. By using a 
+language like this one, we would be able to assign everyone tasks where they know 
+about how long it will take them. Hopefully, this will eliminate the short, 
+stressful periods with longer, calmer periods. This will help both the people and 
+horses._
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+_Organized, prioritized to do lists_
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+_The user should be able to input their to do list based on categories. They
```

Comment:
I'm curious as to what other information would be associated with each task in order for a program to organize the tasks. To-do lists can get very complicated. For example, there could be a series of tasks, each assigned to a different person, that rely on each other. This DSL would need to have a lot of flexibility in terms of expressiveness to allow such customization.